.github/workflows/daily_ci.yml

```diff
@@ -118,3 +118,22 @@ jobs:
           cd $GITHUB_WORKSPACE/zerosoc
           ./build.py --core-only
           ./build.py --top-only
+
+  windows_unit_tests:
+    name: Windows Unit Tests
+    runs-on: windows-latest
+    steps:
+      - name: Checkout SiliconCompiler
+        uses: actions/checkout@v3
+
+      - name: Install Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.8
+          architecture: x64
+
+      - name: Install SC Without Surelog
+        run: READTHEDOCS=True pip3 install -r requirements.txt .
```

Comment:
you could grab the cached version of surelog.
remove the READTHEDOCS=True thing, since this is a hack to avoid running the leflib build. But that should be easy to run.

Reply:
I just did it this way because Surelog isn't needed for a unit test, and functional tests cannot run on windows because the other tools are not compatible.